etna/ciim/constants.py

```diff
@@ -137,7 +133,8 @@ def __iter__(self):
                 Aggregation.CLOSURE,
                 Aggregation.HELD_BY,
                 Aggregation.CATALOGUE_SOURCE,
-                Aggregation.GROUP,
+                Aggregation.GROUP
+                + ":30",  # Fetch more 'groups' so that we receive counts for any bucket/tab options we might be showing (not just the 10 most popular)
```

Comment:
I don't think this needs an explanation (especially not every time)

Reply:
1.  What are the thoughts on why it does not need explanation?  It looks important as per the existing code base [here](https://github.com/nationalarchives/ds-wagtail/blob/4f04108367c81a0b41b801be3dfa3031bdd6e11e/etna/search/views.py#L422-L428) which is why when it was moved here, moved the comment as well.
2. When I put it, I thought its an important value (and not a variable) that must be passed, otherwise make it constant (as it was applied to all buckets in previous code) since it is repetitive. But then you mentioned "_I would say let's avoid adding new constants for this. It's only coincidence that we want 30 groups for each bucket. There is no reason why this shouldn't change in future_." What are the thoughts on not repeating repeating it every time?